accounting/authentication/models.py

```diff
@@ -20,3 +20,29 @@
 """
 Authentication models.
 """
+
+from django.contrib.auth import models
+from django.db.models import CharField
+from django.utils.translation import ugettext_lazy as _
+
+from accounting.common.models import UuidModel
+
+
+class User(UuidModel, models.AbstractUser):
+    """
+    A `User` model with custom columns, like `uuid` and `full_name`.
+    """
+
+    full_name = CharField(
```

Comment:
What some people want as a full name isn't just a first and last name. It may include a middle name, or some other titles and what not that we can easily cover with a catch-all `full_name` field.

Reply:
@UmanShahzad 

> What some people want as a full name isn't just a first and last name.

OK, that's what I thought; was just assuming that we didn't currently have a use case for this as a team. I'm fine with keeping it.